### DISCUSSION TITLE
PnP Search

### DISCUSSION THREAD

**Discussion by Torask** [microsoft-search/pnp-modern-search#3641] [General]
Hello, I would like to ensure that the display settings for the PNP search feature is configured to show information from the "About Me-Competence" field when hovering over the card. Is there option or setting that needs to be adjusted to enable this feature.

**Comment by kasperbolarsen** [comment]
as usually there are multiple options available. If you want to tailor the display template you might look at the sample at https://github.com/microsoft-search/pnp-modern-search-layouts/blob/main/Results/Handlebars/PeopleSearch%20Two%20Columns/PeopleSearchTwoColumns.md
If you want the change to be available all over, also outside PnP Modern Search, you might have a look at https://learn.microsoft.com/en-us/graph/add-properties-profilecard

**Reply by Torask** [reply]
Thank you very much!
